apps/io-func-wallet-solution-backend/src/wallet-instance.ts

```diff
@@ -24,6 +24,18 @@ export type WalletInstanceRepository = {
     userId: WalletInstance["userId"]
   ) => TE.TaskEither<Error, WalletInstance>;
   insert: (walletInstance: WalletInstance) => TE.TaskEither<Error, void>;
+  batchPatch: (
+    operations: Array<{
+      id: string;
+      path: string;
+      value: unknown;
+      op: "add" | "replace" | "remove" | "set" | "incr";
```

Comment:
ditto

Reply:
no we don't need it for now, just as we wouldn't need `value` (which is `true`) and `path` (which is `isRevoked`). It's just that the `patch` requires those parameters. Would you like to remove it?